src/Effect/Exec.hs

```diff
@@ -60,6 +61,9 @@ instance RecordableValue Command
 instance FromJSON Command
 instance ReplayableValue Command
 
+instance ToText Command where
+  toText (Command name args _) = Text.intercalate " " $ [name] <> args
+
```

Comment:
I don't know if I agree with this use of `ToText`.  When used, it will read as `toText cabalCommand`, which doesn't actually tell me how the command is formatted.  I can _kind of_ infer that it's going to be in shell format, but I think this may be better off as something like `renderCommand :: Command -> Text`, instead of using the `ToText` instance.  I think we should only use `ToText` for string-like types.

Reply:
An instance where `ToText` makes more sense is in locators.  Locators are really just strings, but we can access partitions of them via fields like `fetcher` and `revision`.  To do that, we secretly store the string as its component parts, use that to make the field accessors work, and then silently put them back together when we run `ToText`.

Reply:
resolved in: https://github.com/fossas/fossa-cli/pull/803/commits/93cdf4ecf7fb004f155f7337c9c850dd29e1189d